### DISCUSSION TITLE
Concat and Zip

### DISCUSSION THREAD

**Discussion by AureliusHogan** [CarbonPackages/Carbon.IncludeAssets#17] [Ideas]
Are there any plans for concatenating and zipping of CSS files and JS files?

So that there is only one CSS File and one JS File to pull when the user views the site? With an option to configure it for each FLOW_CONTEXT separately?

**Comment by jonnitto** [comment]
I think this should better be done from your Frontend Build setup. As this kind of stuff is static, this would slow down Neos. Also you would lose the benefit of the browser caching, as on every page could be different JS/CSS files. 

BTW: Do you know [Carbon.Pipeline](https://github.com/CarbonPackages/Carbon.Pipeline)?
